server/src-lib/Hasura/Backends/Postgres/Translate/Select/Internal/Process.hs

```diff
@@ -342,10 +342,15 @@ processAnnFields sourcePrefix fieldAlias similarArrFields annFields tCase = do
               fieldName
               PLSQNotRequired
               sel
-          let computedFieldTableSetSource = ComputedFieldTableSetSource fieldName selectSource
+          let selectSourceWithoutNulls =
+                case selectSource of
+                  SelectSource {_ssFrom = S.FIFunc (S.FunctionExp {S.feAlias = Just (S.FunctionAlias {S._faIdentifier = fnIden})})} ->
```

Comment:
@plcplc I think I need some help here, probably there is neater way of getting a function alias for adding it to a `WHERE` clause

Reply:
Don't worry about that - to me this does not look like an unreasonable way to get a function alias.

But is it correct to add a `WHERE function-result-alias IS NOT NULL` condition? It's been a while since I looked at this PR, but if you have a function which might return null (which it can in principle, since it returns a single (if composite) value) have that be reflected in the response you get?